app/src/pages/instance/organizationsPage/organizationsPageHeader/organizationsFilter/organizationsFilter.jsx

```diff
@@ -215,6 +237,10 @@ export const OrganizationsFilter = ({
           initialFilterState[TEAMMATES_FILTER_NAME_CONDITION] && isApply;
     }
 
+    if (customTimeRange) {
+      isApply = customTimeRange === initialFilterState[LAST_RUN_DATE_FILTER_NAME];
```

Comment:
And again here the `LAST_RUN_DATE_FILTER_NAME` from `formValues` can be used to define the sate for `Apply` button.